<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="textEditIn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="tabStopWidth">
           <number>48</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditOut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="tabStopWidth">
           <number>48</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditCompere">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="tabStopWidth">
           <number>48</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextEdit" name="textEditInDelays">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="tabStopWidth">
           <number>48</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditOutDelays">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="tabStopWidth">
           <number>48</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditCompereDelays">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="tabStopWidth">
           <number>48</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="settingBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Настиройки вида</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="charView">
           <item>
            <property name="text">
             <string>HEX</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UTF-8</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pushButtonStart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Отправить по...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSending">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Таймаут</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditTimeout">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupIn">
        <property name="title">
         <string>Переатчик</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButtonIn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Включить передатчик</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelInOpen">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Off</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonEthIn">
           <property name="text">
            <string>Ethernet</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonRemoteEthIn">
           <property name="text">
            <string>Удаленный Ethernet</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupOut">
        <property name="title">
         <string>Приемник</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButtonOut">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Включить приемник</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelOutOpen">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Off</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonEthOut">
           <property name="text">
            <string>Ethernet</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonRemoteEthOut">
           <property name="text">
            <string>Удаленный Ethernet</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonEnd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Завершить все сеансы</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Интерфейсы</string>
    </property>
    <addaction name="actionEhtTSettings"/>
    <addaction name="actionEhtRSettings"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Кадры</string>
    </property>
    <addaction name="actionFrame"/>
    <addaction name="actionSaveAll"/>
    <addaction name="actionSaveIn"/>
    <addaction name="actionSaveOut"/>
    <addaction name="actionOpenIn"/>
    <addaction name="actionOpenOut"/>
    <addaction name="actionOpenAll"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFrame"/>
   <addaction name="actionSaveAll"/>
   <addaction name="actionOpenAll"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFrame">
   <property name="text">
    <string>Открыть папку</string>
   </property>
   <property name="toolTip">
    <string>Настройка кадров передоваемых и получаемых с устройства</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>Сохранить все</string>
   </property>
   <property name="toolTip">
    <string>Сохранить входные и выходные файлы</string>
   </property>
  </action>
  <action name="actionUsbSettings">
   <property name="text">
    <string>Настройка USB</string>
   </property>
   <property name="toolTip">
    <string>Выбор и настройка USB портов и устройств</string>
   </property>
  </action>
  <action name="actionEhtTSettings">
   <property name="text">
    <string>Настройка Ethernet передатчика</string>
   </property>
   <property name="toolTip">
    <string>Настройка Ethernet передатчика</string>
   </property>
  </action>
  <action name="actionEhtRSettings">
   <property name="text">
    <string>Настройка Ethernet приемника</string>
   </property>
   <property name="toolTip">
    <string>Настройка Ethernet приемника</string>
   </property>
  </action>
  <action name="actionEthStart">
   <property name="text">
    <string>Передача по Eth</string>
   </property>
   <property name="toolTip">
    <string>Начать передачу по Ethernet</string>
   </property>
  </action>
  <action name="actionUSBStart">
   <property name="text">
    <string>Передача по USB</string>
   </property>
   <property name="toolTip">
    <string>Начать передачу по USB</string>
   </property>
  </action>
  <action name="actionFrameIn">
   <property name="text">
    <string>Входной кадр</string>
   </property>
   <property name="toolTip">
    <string>Настройка кадра получаемого с устройства</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Передача по...</string>
   </property>
   <property name="toolTip">
    <string>Начать передачу по настраевыемым интерфейсам</string>
   </property>
  </action>
  <action name="actionSaveIn">
   <property name="text">
    <string>Сохранить файлы передачи</string>
   </property>
   <property name="toolTip">
    <string>Сохранить файлы отправляемые на устройство</string>
   </property>
  </action>
  <action name="actionSaveOut">
   <property name="text">
    <string>Сохранить файлы приема</string>
   </property>
   <property name="toolTip">
    <string>Сохранить файлы получаемые с устройства</string>
   </property>
  </action>
  <action name="actionOpenIn">
   <property name="text">
    <string>Загрузить  файлы передачи</string>
   </property>
   <property name="toolTip">
    <string>Открыть файлы отправляемые на устройство</string>
   </property>
  </action>
  <action name="actionOpenOut">
   <property name="text">
    <string>Загрузить файлы приема</string>
   </property>
   <property name="toolTip">
    <string>Загрузить файлы получаемые с устройства</string>
   </property>
  </action>
  <action name="actionUpdateIn">
   <property name="text">
    <string>Обновить передаваемые данные</string>
   </property>
   <property name="toolTip">
    <string>Сохранить текущие изменения передоваемых данных и подготовить их к отправке</string>
   </property>
  </action>
  <action name="actionOpenAll">
   <property name="text">
    <string>Загрузить все</string>
   </property>
   <property name="toolTip">
    <string>Загрузить входные и выходные файлы</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
